src/main/java/org/fundacion/jala/converter/service/javacompiler/JavaVersion.java

```diff
@@ -5,11 +5,16 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Jessicka Moya Andrade
  */
 package org.fundacion.jala.converter.service.javacompiler;
 
+/**
+ * This enum defines possibles Java versions.
```

Comment:
It is "possible" intead of "possibles "

Reply:
You are rigth, I have corrected to "possible".